internal/corazawaf/transaction.go

```diff
@@ -1387,6 +1389,7 @@ func (tx *Transaction) AuditLog() *auditlog.Log {
 			Method_:   tx.variables.requestMethod.Get(),
 			URI_:      tx.variables.requestURI.Get(),
 			Protocol_: tx.variables.requestProtocol.Get(),
+			Args_:     tx.variables.args,
```

Comment:
`Args()`, `Length()`, and `UID()` have been added to the `AuditLogTransactionRequest` interface. The OCSF Formatter then reads these values, but here I see that only `Args_` is populated. Also, in the test, I don't see their value being checked. As far as I get this, Length() and UID() will always just return the default (0 and "") values. Should also these values be populated here?